src/components/treeCard/index.tsx

```diff
@@ -44,12 +51,21 @@ const TreeCard: React.FC<TreeCardProps> = ({ site }) => {
           </CardInfo>
           <GreenLinkButton
             to={`${ParameterizedRouteBases.TREE}${site.id}`}
-            state={{
-              destination: Routes.MY_TREES,
-            }}
+            state={{ destination: Routes.MY_TREES }}
           >
             More Info
           </GreenLinkButton>
+
+          {userIsAdmin && (
+            <GreenLinkButton
+              to={`${ParameterizedRouteBases.SITE}${site.id}`}
+              state={{ destination: Routes.MY_TREES }}
+              target="_blank"
+              style={{ marginLeft: '.1rem' }}
```

Comment:
⛏️ Prefer to avoid inline styling here too. You can make that StyledDiv a theme component and reuse it here (prob rename to something a bit more specific)